hotnews/index.html

```diff
@@ -1,37 +1,107 @@
 <html>
-  <head>
-    <title>HotNews.ro</title>
-    <link rel="icon" type="image/png" sizes="32x32" href="./assets/favicon.png">
 
-    <script src="https://cdn.tailwindcss.com"></script>
-    <script>
-      tailwind.config = {
-        theme: {
-          extend: {
-            colors: {
-              hotblue: '#2145ac',
-              hotgreen: '#3f9863',
-              hotred: '#eb3223'
-            }
+<head>
+  <title>HotNews.ro</title>
+  <link rel="icon" type="image/png" sizes="32x32" href="./assets/favicon.png">
+  <link rel="stylesheet" href="hotnews.css">
+
+  <script src="https://cdn.tailwindcss.com"></script>
+  <script>
+    tailwind.config = {
+      theme: {
+        fontSize: {
+          font: '15px',
+          font2: '18px',
+          font3: '14px'
```

Comment:
I tried text-sm which is 14px but doesn't work and that's why i defined custom 14px.
The texts where you will see the difference are: "Actualizat acum un minut and EUR 4.9353 USD 4.8509"
Here you have one photo with text-sm:

![Screenshot 2022-07-29 at 14 01 26 (2)](https://user-images.githubusercontent.com/103902373/181746320-58103879-7839-4f26-8f19-a3a61a762796.png)

Here you have one photo with custom 14px:

![Screenshot 2022-07-29 at 14 06 31](https://user-images.githubusercontent.com/103902373/181746909-57bcd191-efff-4829-8e7c-b3d940dcc493.png)

Reply:
@Singeorzan97 it's because you have overriden the default tailwind font sizes with your options

you should instead use the following syntax:

extend: {
  fontSize: {
    font: '15px',
    font2: '18px'
  },
  width: {
    976: '976px',
  },
  ...  
}

Reply:
I've tried again and it's the same, doesn't work.